File: OpenTokFs.ResponseTypes/OpenTokStream.cs
using System.Collections.Generic;
using System.Linq;

namespace OpenTokFs.ResponseTypes {
    public class OpenTokStream {
        public string Id { get; set; } = "";
        public string VideoType { get; set; } = "";
        public string? Name { get; set; } = null;
        public IEnumerable<string> LayoutClassList { get; set; } = Enumerable.Empty<string>();

        public bool IsCamera => VideoType == "camera";
        public bool IsScreen => VideoType == "screen";

        public override string ToString() => $"{Id} ({Name ?? "no name"}) ({VideoType})";
    }
}

File: OpenTokFs/ResponseTypes/OpenTokStream.fs
namespace OpenTokFs.ResponseTypes

open System

type OpenTokStream = {
    id: string
    videoType: string
    name: string // or null
    layoutClassList: string list
} with
    member this.IsCamera = this.videoType = "camera"
    member this.IsScreen = this.videoType = "screen"
    override this.ToString () = String.concat " " [
        this.id
        if not (isNull this.name) then this.name
        this.videoType
    ]